src/lib/server/trigger.c

```diff
@@ -461,15 +461,15 @@ int trigger_exec(request_t *request, CONF_SECTION const *cs, char const *name, b
  *	- NULL on failure, or if triggers are not enabled.
  *	- list containing Pool-Server and Pool-Port
  */
-fr_pair_t *trigger_args_afrom_server(TALLOC_CTX *ctx, char const *server, uint16_t port)
+fr_pair_list_t *trigger_args_afrom_server(TALLOC_CTX *ctx, char const *server, uint16_t port)
```

Comment:
Another one where we should really be passing in the list and adding the pairs to it